posthog/api/plugin.py

```diff
@@ -237,6 +241,8 @@ class PluginOwnershipPermission(BasePermission):
     message = "This plugin installation is managed by another organization."
 
     def has_object_permission(self, request, view, object) -> bool:
+        if request.method in SAFE_METHODS:
```

Comment:
This _feels_ off but I'm struggling to figure out why. I think there might be a better way but I would need to get into the weeds of what this is all supposed to do. Maybe something we can pair on if we want to be sure we don't introduce some accidental leakage